src/EnergyPlus/DualDuct.cc

```diff
@@ -2337,6 +2339,16 @@ namespace DualDuct {
         }
     }
 
+    void DualDuctAirTerminal::CalcOutdoorAirVolumeFlowRate()
+    {
+        // calculates the amount of outdoor air volume flow rate using the supply air flow rate and OA fraction
+        if (this->AirLoopNum > 0) {
+            this->OutdoorAirFlowRate = (this->dd_airterminalOutlet.AirMassFlowRate / StdRhoAir) * DataAirLoop::AirLoopFlow(this->AirLoopNum).OAFrac;
+        } else {
+            // do nothing for now
+        }
+    }
+
```

Comment:
the new member common function added for use with AirTerminal:DualDuct:VAV and AirTerminal:DualDuct:ConstantVolume objects.

Reply:
Will there ever be anything in the ELSE block here?